scripts/upgrade

```diff
@@ -17,6 +17,9 @@ sudo yunohost app upgrade $app -u https://github.com/YunoHost-Apps/gitea_ynh/com
     ynh_die "$message" --ret_code=1
 fi
 
+# If indexer doesn't exist, create it
+ynh_app_setting_set_default --app=$app --key=repos_indexer_enabled --value=false
```

Comment:
Maybe you can just set the variable here https://github.com/YunoHost-Apps/gitea_ynh/blob/e028b14f379f052f790fa64c545e231703b6c4e5/scripts/_common.sh#L27 for install/upgrade so we set everything at the same place.